docs/user-guide/cli-using-global-profile-configuration.md

```diff
@@ -0,0 +1,535 @@
+# Using Global Profile Configuration (Technical Preview) <!-- omit in toc -->
```

Comment:
In various presentations we have been using the term "team configuration" instead of "global profile configuration". I do not think that this needs to be resolved before publishing this @next documentation. However, we may want to decide on consistent terminology before we deliver GA doc in V2-LTS.

Reply:
@MikeBauerCA Any input on this?